### ISSUE TITLE
/place template causes error during validation

### ISSUE THREAD

**Issue by 2mal3** [mcbeet/mecha#167]
Validating a datapack (e.g. [pack.zip](https://github.com/mcbeet/mecha/files/9174424/pack.zip)) with a `place template` command (e.g. `place template minecraft:fossil/spine_1 ~ ~ ~`) leads to the following error:
```
Error: Plugin "mecha.cli.validate" raised an exception.

Traceback (most recent call last):
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/cli.py", line 30, in validate
    mc.compile(DataPack(path=path), report=mc.diagnostics)
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/api.py", line 432, in compile
    compilation_unit.ast = self.parse(
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/api.py", line 303, in parse
    ast = delegate(parser, stream)
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/parse.py", line 532, in delegate
    return spec.parsers[parser](stream)
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/parse.py", line 573, in parse_root
    commands.append(delegate("command", stream))
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/parse.py", line 532, in delegate
    return spec.parsers[parser](stream)
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/parse.py", line 638, in parse_command
    argument = delegate("command:argument", stream)
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/parse.py", line 532, in delegate
    return spec.parsers[parser](stream)
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/parse.py", line 692, in parse_argument
    return delegate(f"command:argument:{tree.parser}", stream)
  File "/home/user/Desktop/pack/venv/lib/python3.10/site-packages/mecha/parse.py", line 530, in delegate
    raise UnrecognizedParser(parser)
mecha.parse.UnrecognizedParser: command:argument:minecraft:template_rotation
```
I have noticed that this does not happen if you leave out the **~** at the end (only `place template minecraft:fossil/spine_1`).

**Comment by vberlier** [comment]
Ooh good catch! Looks like we're missing an argument parser. I should be able to fix this later.

**Comment by vberlier** [comment]
@2mal3 Just fixed it in https://github.com/mcbeet/mecha/commit/718c907e263e3b516fe1881fd3f4775485d3d1e9. This is released in `v0.54.8`. If you're using the `check-commands` action you can bump `mecha` with the `version` input.
